src/config.c

```diff
@@ -285,7 +285,8 @@ static void resetConf(void)
     cfg.D8[PIDVEL] = 1;
     cfg.rcRate8 = 90;
     cfg.rcExpo8 = 65;
-    cfg.rollPitchRate = 0;
+    cfg.rollPitchRate[0] = 0;
+    cfg.rollPitchRate[1] = 0;

```

Comment:
not really needed, array is already zero'd anyway.
old stuff stays, but new zero vars shouldn't be added.